events/codexClient.go

```diff
@@ -93,21 +111,16 @@ func buildGetRequest(address string, auth acquire.Acquirer) (*http.Request, erro
 	return request, nil
 }
 
-func circuitBreakerRequestFunc(c *CodexClient) func(req *http.Request) (*http.Response, error) {
-	return func(req *http.Request) (*http.Response, error) {
-		resp, err := c.CB.Execute(func() (interface{}, error) {
-			return c.Client.Do(req)
-		})
-
-		if err != nil {
-			return nil, err
+// logs prometheus metrics when circuit breaker state changes
+func onStateChanged(m Measures) func(string, gobreaker.State, gobreaker.State) {
```

Comment:
this is awesome. 🚀 

A gauge might make more sense for `CircuitBreakerOpenCount`
1 -> Open, 0.5 -> Half-open 0 -> closed